ci/Jenkinsfile.combined

```diff
@@ -25,10 +25,14 @@ pipeline {
     stage('Build') {
       parallel {
         stage('MacOS') { steps { script {
-          osx = cmn.buildBranch('status-react/combined/desktop-macos', cmn.getBuildType())
```

Comment:
This is not how your use decarative pipelines:
https://jenkins.io/doc/book/pipeline/syntax/#when
So in this case it would be something like:
```
        stage('MacOS') {
          when { expression { cmn.getBuildType() != 'release' } }
          steps { script {
            osx = cmn.buildBranch('status-react/combined/desktop-macos', cmn.getBuildType())
        } } }
```